Resources/public/js/cookie_consent.js

```diff
@@ -37,7 +37,13 @@ document.addEventListener("DOMContentLoaded", function() {
                 xhr.open('POST', cookieConsentForm.action);
                 xhr.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded');
                 xhr.send(serializeForm(cookieConsentForm, event.target));
-		// Clear all styles from body to prevent the white margin at the end of the page
+
+                let buttonEvent = new CustomEvent("cookie-consent-form-button-click", {
```

Comment:
You could use `var` instead of `let` to keep it consistent with the other variables.